src/util/parse_options.h

```diff
@@ -35,6 +35,17 @@ class parse_options_baset
   bool parse_result;
 };
 
+class parse_optionst : public parse_options_baset
```

Comment:
I'm not sure we need this derived class: can't it just be changed in `parse_options_baset` and make everyone benefit without changes?

Reply:
This is not needed anymore. I have eliminated it.